backends/gstreamer/src/player.rs

```diff
@@ -20,6 +20,7 @@ use std::time;
 use std::u64;
 
 const MAX_SRC_QUEUE_SIZE: u64 = 50 * 1024 * 1024; // 50 MB.
+const MAX_BUFFER_SIZE: i32 = 500 * 1024;
```

Comment:
why this magic number?

Reply:
I got it from Gecko. I guess we could check disk space and make this more dynamic, but I'd prefer to do that in a follow up.